projects/WeTek_Core/initramfs/platform_init

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 ################################################################################
 #      This file is part of OpenELEC - http://www.openelec.tv
-#      Copyright (C) 2009-2016 Stephan Raue (stephan@openelec.tv)
```

Comment:
I took the init_script from the WeTek_Play project adapted it and copied it also to the WeTek_Core project. So what do you propose to do?